javaScript-week1/Housey-pricey.js

```diff
@@ -0,0 +1,30 @@
+// Housey pricey
+
+function sumVolume(width, height, depth) {
+    return width * height * depth;
+  }
+  
+  function trueHousePrice(volumeInMeters, gardenSize) {
+    return volumeInMeters * 2.5 * 1000 + gardenSize * 300;
+  }
```

Comment:
Great job at using functions and making two, each for calculating a specific thing.